### ISSUE TITLE
Suspicious results

### ISSUE THREAD

**Issue by FiReTiTi** [pytorch/vision#2004]
## Questions about suspicious results ❓

I've trained MaskRCNN with a pre-trained ResNet50 to segment nuclei in immunofluorescence images. The results are really good, so thanks again for this terrific implementation.

However, I've noticed in some cases that an object (here a nucleus) might be cut in multiple small parts (see bottom right part of the attached image).

<img width="531" alt="Capture d’écran 2020-03-23 à 15 27 27" src="https://user-images.githubusercontent.com/6014800/77369519-0fcfa600-6d1c-11ea-820d-3186fc1bc037.png">

We can observe that the nucleus labeled 247 is cut in multiple parts.
I get that in most detection/segmentation applications, an object can partially obfuscate another one that is further in a scene. But, is it a normal behavior for this implementation?

**Comment by fmassa** [comment]
Hi,

This could happen with Mask R-CNN (and other detection) models. What probably happened is that the object was split in multiple detections at the box prediction step, and the segmentation head did it's best given the splitted boxes. But I might be wrong :-)

Given that there is nothing actionable that we can do here, I'm closing the issue, but let us know if you have further questions

**Comment by FiReTiTi** [comment]
Thank you for your answer.
I have single bounding box as a result, not multiple, this is why I was surprised.

**Comment by fmassa** [comment]
So, each segmentation mask is associated with a single bounding box, so it would be good to visualize the boxes individually to try to understand what is going on

**Comment by FiReTiTi** [comment]
I didn't save the bounding boxes for this example :-(
But the three pieces that represent object 147 were all in the same box.
